### PR TITLE
fix map tooltip hover

### LINKED ISSUE #136: Difficult to target "Learn More" link and any small state on the hovercard
It's currently difficult to reach the "Learn More" link on the hovercard, as well as to reach any small state, because if that part of the hovercard is over a different state, the card will reload with that other state's information. It would be better if the card was frozen while the user's cursor is over it.

See:
https://user-images.githubusercontent.com/6788487/110365331-c2fee100-7ff9-11eb-9b46-9c0b2a9e5299.mov

### PR DESCRIPTION
**Pull request by landonreed**
Closes #136.

### CODE REVIEW

File: components/UsMap.js [hunk 1]
```diff
@@ -149,24 +151,23 @@ class UsMap extends Component {
             //     .style("opacity", 0)
             // }
             // only redraw the tooltip if we're NOT in the bounds of a tool tip card
-            console.log(mouseInsideTooltip());
+            // console.log(mouseInsideTooltip(document.getElementById('tooltip').getBoundingClientRect()));
             // click events
         }).on("click", function (d) {
             // Navigate to state on click
             // FIXME: replace with toSlug method from utils
```
Comment (egiurleo) [inline components/UsMap.js]: Is this FIXME irrelevant now?

**Review by egiurleo — APPROVED** [review]
Thanks for fixing!

File: components/UsMap.js (hunk 1 above)
Comment (landonreed) [reply components/UsMap.js]: Woops yep!

**Review by sarahgreen — APPROVED** [review]
LGTM!! Thanks!

**Pull request merged**